github_connector_odoo/models/odoo_module_version.py

```diff
@@ -103,16 +103,28 @@ class OdooModuleVersion(models.Model):
 
     author = fields.Char(string='Author (Manifest)', readonly=True)
 
+    maintainer = fields.Char(string='Maintainer (Manifest)', readonly=True)
+
     author_ids = fields.Many2many(
         string='Authors', comodel_name='odoo.author',
         relation='github_module_version_author_rel',
         column1='module_version_id', column2='author_id', multi='author',
         compute='_compute_author', store=True)
 
+    maintainer_ids = fields.Many2many(
+        string='Maintainers', comodel_name='odoo.maintainer',
```

Comment:
I also think this should be a m2m to the res.partner with corresponding github login.